src/app/components/Promotions/index.tsx

```diff
@@ -49,6 +55,18 @@ export const Promotions: React.FC<PromotionsProps> = ({
             linkDataActionId={`landing-promo-learnmore-${Asset.DLLR}`}
             className={cardClassName}
             imageClassName={cardImageClassName}
+            ammData={
+              ammData &&
+              ammData[
+                LiquidityPoolDictionary.get(
+                  Asset.DLLR,
+                  Asset.RBTC,
+                ).converter.toLowerCase()
+              ]
```

Comment:
you could have some helper function that accepts `ammData` and both assets and returns this